pages/home.md

```diff
@@ -14,14 +14,14 @@ permalink: /
         />
       </figure>
       <p class="subtitle has-text-centered is-size-4 is-size-3-tablet is-size-2-desktop">
-      Using Web standards to
-      let people control their data, and choose the
-      applications and services to use with it.
+      Developing Web standards that
+      enables people to control their data. 
       </p>
       <div class="buttons is-centered">
-        <a href="{{site.baseurl}}/users/get-a-pod" class="button is-black is-large">
-          Get a Pod
+        <a href="{{site.baseurl}}/about" class="button is-primary  is-large"> 
+          Learn about Solid
```

Comment:
Changed the button per discussion to the About page.

I moved the Pod button below (but, I also tried it as the middle button and it wasn't bad either).

Reply:
Learn more Button might look a bit better and stand out against the other one with an alternate color